receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_linux.go

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build windows
```

Comment:
Make Linux the special case instead of Windows, as other systems including Darwin seem to use the same standard names as Windows.